.config/.env.example

```diff
@@ -10,3 +10,11 @@ MONGO_PORT=
 MONGO_DB=logs
 MONGO_USER=admin
 MONGO_PASSWORD=
+
+# KEYCLOAK_USER=
```

Comment:
Not sure I like commented out vars in the example .env file. It's not obvious when they may be needed. Can you explain the purpose of this @mauriciofreshw?

Reply:
This env var is being used in the keycloak container that’s also commented out. So if someone intends to use keyclaok fully locally those env var will be necessary.

Reply:
The .env is really only used locally right? Seems like we could uncomment them